src/code42cli/cmds/search/extraction.py

```diff
@@ -124,3 +129,34 @@ def create_time_range_filter(filter_cls, begin_date=None, end_date=None):
 
     elif end_date and not begin_date:
         return filter_cls.on_or_before(end_date)
+
+
+def create_send_to_handlers(
+    sdk, extractor_class, cursor_store, checkpoint_name, logger
+):
+    extractor = extractor_class(sdk, ExtractionHandlers())
+    handlers = _set_handlers(cursor_store, checkpoint_name)
+
+    @warn_interrupt(warning=INTERRUPT_WARNING)
+    def handle_response(response):
+        response_dict = json.loads(response.text)
+        events = response_dict.get(extractor._key)
+        if extractor._key == "alerts":
```

Comment:
Lines 144 - 148 seems like they can be moved to a separate function and called by both `create_handlers()` methods